### ISSUE TITLE
Radiant Player keeps telling me I need to update Adobe Flash Player

### ISSUE THREAD

**Issue by sritter617** [radiant-player/radiant-player-mac#540]
I have my default browser set to Google Chrome and I have Adobe Flash Player installed and up to date in Safari and Radiant Player is still telling me to upgrade Adobe Flash. I don't know what else to do and no other forum has been helpful. Please Help!!!

**Comment by jacobwgillespie** [comment]
What version of Radiant and OSX do you have?

**Comment by sritter617** [comment]
Radiant Player 1.8.2 and OSX 10.11.3

**Comment by noone1337** [comment]
I have the same problem with OSX 10.11.4... Radiant Player 1.8.3

**Comment by mradzinski** [comment]
Same over here

**Comment by SourceCode** [comment]
I have the same issue - Radiant Player 1.8.3 on OS X 10.11.4 - seems to tell me I need flash once a week or after a restart of my machine.  I can even have been listening to music and then close Radiant, re-open it and it says I need to install flash.  I had our IT department check it out to see if it was some kind of security policy as we use a OS X security suite for anti-virus and policies and it turns out we don't have any of that in place (from what they could tell).  Thank you for all the hard work on this project - it's the best player!

**Comment by gravyplaya** [comment]
With Chrome completely disabling Flash can you please look into a HTML5 implementation perhaps? Because otherwise this app essentially dies with Chrome Flash support. Cant let that happen. :)

**Comment by nowthatsmint** [comment]
Have you guys tried **downloading Flash from Safari** ? this worked for me — I'm on El Capitan and originally downloaded Flash from Chrome and that did nothing.

**Comment by jacobwgillespie** [comment]
Yeah, make sure you install Flash from Safari as that's what Radiant uses for the backend at the moment.

Tangentially, I've been working on [@radiant-player/radiant-player-electron](https://github.com/radiant-player/radiant-player-electron) which I'm hoping will become the next version of Radiant.  The goal is to completely replicate the existing look / feel and feature set of Radiant 1.x, and is Chrome-based so HTML5 playback will be supported.  It is currently pre-alpha at the moment though.

tl;dr - install Flash in Chrome, I'm investigating removing the flash requirement.

**Comment by iandouglas** [comment]
I'm on Radiant 1.11.2 and I get this prompt all the time as well. I downloaded an installer from Adobe, installed Flash, restarted Radiant and still get the error. Even trying to install Flash through Safari gives me the same steps from Adobe:
```1. To proceed, open your download folder and locate the Adobe Flash Player installer file, for example “AdobeFlashPlayer[xxx]_install.dmg”.
2. Double-click on the installer to complete the installation. For additional help, click here.```

Flash downloads, installs, and Radiant still doesn't work. The player was highly recommended from a colleague, so I'd love to try it out.

**Comment by jkmaxwell** [comment]
any update on the electron front?
